.github/workflows/release_crates.yml

```diff
@@ -0,0 +1,21 @@
+name: Release crates
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 2-6'
+  push:
+    branches:
+      - main
+
+jobs:
+  environment: carte-publish
```

Comment:
I added this as well as other release workflows. I don't have the permission to add, so I would like to have it added.